includes/abstracts/abstract-wc-gateway-ppec.php

```diff
@@ -532,8 +532,7 @@ public function generate_image_html( $key, $data ) {
 		?>
 		<tr valign="top">
 			<th scope="row" class="titledesc">
-				<?php echo $this->get_tooltip_html( $data ); ?>
-				<label for="<?php echo esc_attr( $field_key ); ?>"><?php echo wp_kses_post( $data['title'] ); ?></label>
+				<label for="<?php echo esc_attr( $field_key ); ?>"><?php echo wp_kses_post( $data['title'] ); ?> <?php echo $this->get_tooltip_html( $data ); ?></label>
```

Comment:
Nitpick: does the tooltip require a leading space? Also, you can combine the `echo` statements with commas.

Reply:
Good feedback – I think I'll leave it as it is, to align with https://github.com/woocommerce/woocommerce/commit/d8814f93819a41a64f9f6571542488de8db65dab.